### PR TITLE
Change font

### PR DESCRIPTION
**Pull request by hayeonhee**
## Changes
- Change font
- Fix padding that was not applied

### CODE REVIEW

**Comment by sebatyler** [comment]
Task linked: [CU-33e34xz Change font](https://app.clickup.com/t/33e34xz)

File: lib/widgets/common.dart [hunk 1]
```diff
@@ -102,14 +102,14 @@ mixin CommonWidgetMixin {
...
-        crossAxisAlignment: CrossAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.center,
         children: [
           Text(
             tag,
             style: TextStyle(
               fontWeight: FontWeight.w400,
               fontSize: 14.0,
-              height: 1.57,
+              // height: 1.57,
```
Comment (sebatyler) [inline lib/widgets/common.dart]: If this is not required, why don't you remove this line?

Comment (hayeonhee) [reply lib/widgets/common.dart]: Thanks, I removed the line.

**Review by sebatyler — APPROVED** [review]
Confirmed!

**Comment by hayeonhee** [comment]
Thank you!

**Pull request merged**